apps/lfx-one/.env.example

```
@@ -131,6 +131,17 @@ TEST_PASSWORD=your-test-password
 
 LOG_LEVEL=info
 
+API_GW_AUDIENCE=https://api-gw.dev.platform.linuxfoundation.org/
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)